connector_prestashop/__manifest__.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2011-2013 Camptocamp
+# Copyright 2011-2013 Akretion
+# Copyright 2015 AvanzOSC
+# Copyright 2015-2016 Tecnativa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "PrestaShop-Odoo connector",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": [
+        "account",
+        "base_vat",  # for vat validation on partner address
+        "product",
+        "product_multi_category",  # oca/product-attribute
+        "product_multi_image",  # oca/product-attribute
+        "connector_ecommerce",  # oca/connector-ecommerce
+        "purchase",
+        "onchange_helper",
+        "product_brand",  # oca/product-attribute
```

Comment:
@hailangvn hailangvn
First, thanks for the huge work on the module.
I'll try do do a full review in the future, just a remark for now : 

I believe the connector_prestashop is already really big with a lot of dependencies, we should try not to add anymore.
Indeed, the design is made to make the module quite easily overridable and I think feature likes brand/manufacturer should be in a separate module.
Not every body will use this feature so best to keep in separated to ease further migrations of the base module.

Of course, there are other stuff that could be part of a separated module (refund import for instance ?) but it is in the main module historically.

For instance, the last new feature I needed, the import of product of type "pack", I've made in a separated module (https://github.com/akretion/connector-prestashop/tree/14-connector_prestashop_import_bom)
For the same reason, not every body needs it and it is best for maintenance to keep it separated.

Would you mind putting the brand stuff in an other module ?